config/locales/client.en.yml

```
@@ -32,6 +32,7 @@ en:
         reminder:
           label: Reminder
           description: How often a reminder notification to accept the policy should be sent
+          no-reminder: (no reminder)
```

```suggestion
          no_reminder: (No reminder)
```

Should not use dashes in these yml keys, and also makes the text follow https://meta.discourse.org/t/formatting-text-in-discourse-documentation-and-uis/324637

---

Yes but those are formatted with our our old "lowercase all the things" style we had for years, the meta docs are what is right now. Will open a quick PR here to update the rest

---

https://github.com/discourse/discourse-policy/pull/169